libafl/src/corpus/cached.rs

```diff
@@ -147,6 +147,12 @@ where
         self.inner.next(idx)
     }
 
+    /// Peek the next free corpus id
+    #[inline]
+    fn peek_next_free_id(&self) -> CorpusId {
```

Comment:
just `peek_free_id` is probably precise enough, what do you think? Change it or leave it? @R9295